src/components/TeachersPage/LessonForm/LessonForm.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react'
+
+const LessonForm: React.FC = () => {
+  const [formData, setFormData] = useState({
+    fullName: '',
+    email: '',
+    phone: '',
+    preferences: {
+      careerAndBusiness: false,
+      lessonForKids: false,
+      livingAbroad: false,
+      examsAndCoursework: false,
+      cultureTravelHobby: false
+    }
+  })
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.target
+    setFormData((prev) => ({
+      ...prev,
+      [name]: value
+    }))
+  }
+
+  const handleCheckboxChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, checked } = e.target
+    setFormData((prev) => ({
+      ...prev,
+      preferences: {
+        ...prev.preferences,
+        [name]: checked
+      }
+    }))
+  }
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault()
+    console.log('Form Data:', formData)
+  }
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <div>
+        <label>
+          <input
+            type="checkbox"
+            name="careerAndBusiness"
+            checked={formData.preferences.careerAndBusiness}
+            onChange={handleCheckboxChange}
+          />
+          Career and business
+        </label>
```

Comment:
do custom label + its not checkbox, change to radio

Reply:
custom label are good for use with text input, but i need to fix types for a checkbox(that can be a little complicated), so i want to leave a chackbox like this only in this case if i can